Pod/Classes/PINRemoteImageManager.m

```diff
@@ -598,6 +615,7 @@ - (NSUUID *)downloadImageWithURL:(NSURL *)url
              BOOL taskExisted = NO;
              if (task == nil) {
                  task = [[taskClass alloc] init];
+                 task.retryStrategy = self.retryStrategyCreationBlock();
```

Comment:
Seems like this should either be lazily created when a retry needs to be evaluated or at the very least created as part of the init?